felix/bpf-gpl/tc.c

```diff
@@ -92,6 +93,14 @@ static CALI_BPF_INLINE int calico_tc(struct __sk_buff *skb)
 	}
 	__builtin_memset(ctx.state, 0, sizeof(*ctx.state));
 
+	if (!ctx.counters) {
+		CALI_DEBUG("Counters map lookup failed: DROP\n");
+		// We don't want to drop packets just because counters initialization fails, but
+		// failing here normally should not happen.
+		return TC_ACT_SHOT;
+	}
+	COUNTER_INC(&ctx, CALI_REASON_UNKNOWN); // This is used to keep the total number of packets
```

Comment:
```suggestion
	COUNTER_INC(&ctx, COUNTER_TOTAL_PACKETS);
```

Let the code to speak for itself ;-)

Reply:
Fixed. Thanks

Reply:
Fixed.